src/components/accordion-item/accordion-item.tsx

```diff
@@ -127,23 +127,22 @@ export class AccordionItem {
 
     const { guid } = this;
     const regionId = `${guid}-region`;
-    const buttonId = `${guid}-button`;
 
     return (
-      <Host tabindex="0">
+      <Host>
         <div
+          aria-expanded={toAriaBoolean(this.active)}
           class={{
             [`icon-position--${this.iconPosition}`]: true,
             [`icon-type--${this.iconType}`]: true
           }}
+          onClick={this.itemHeaderClickHandler}
+          role="button"
+          tabindex="0"
         >
           <div
             aria-controls={regionId}
```

Comment:
Do we still need this `aria-controls`?

Reply:
With the earlier updates, we don't. Good catch!